sql-plugin/src/main/330cdh/scala/org/apache/spark/sql/hive/cdh/HiveOverrides.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2019-2023, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.sql.hive.cdh
+
+import com.nvidia.spark.rapids.{ExecChecks, GpuExec, GpuOverrides, SparkPlanMeta, TypeSig}
+
+import org.apache.spark.sql.hive.execution.HiveTableScanExec
+
+object HiveOverrides {
+
+  def getHiveTableExecRule() = {
```

Comment:
We should probably mention that this is a workaround for the mismatches listed in #7423.